src/authorizations/users.rs

```diff
@@ -0,0 +1,42 @@
+fn authorized_users() -> Vec<String> {
+    dotenv::var("AUTHORIZED_USERS")
+        .unwrap()
+        .split(',')
+        .map(String::from)
+        .collect()
+}
+
+pub fn is_authorized(id: String) -> bool {
```

Comment:
This should probably take an `&str` – there is no reason why checking whether a user is authorized should consume the string.